src/core/src/consensus_state_store.rs

```diff
@@ -0,0 +1,121 @@
+/*

```

Comment:
Consensus state store should've contained the functionalities for  storing consensus related info. See:  https://github.com/hyperledger/sawtooth-core/blob/8085604594105ccc478072cafbd1bafe1f4948d6/consensus/poet/core/sawtooth_poet/poet_consensus/consensus_state_store.py

Reply:
Please update the consensus state store to store consensus related info instead of blocks.